apps/emqx_bridge_http/src/emqx_bridge_http_connector.erl

```diff
@@ -236,7 +236,8 @@ on_start(
         port => Port,
         connect_timeout => ConnectTimeout,
         scheme => Scheme,
-        request => preprocess_request(maps:get(request, Config, undefined))
+        request => preprocess_request(maps:get(request, Config, undefined)),
+        installed_actions => #{}
```

Comment:
`on_query` and `on_query_async` for action queries pattern match on this key, which as previously absent if the connector restarted.  It led to the "method" of the request becoming the action resource id...  :melting_face: